src/components/Cluster/NewCluster/CreateNodePoolsCluster.js

```diff
@@ -234,49 +262,82 @@ class CreateNodePoolsCluster extends Component {
                     <InputGroup>
                       <RadioInput
                         id='automatic'
-                        checked={!hasAZLabels}
+                        checked={masterAZMode === MASTER_AZ_MODE_AUTO}
                         label='Automatic'
-                        onChange={() => this.toggleMasterAZSelector(false)}
+                        onChange={() =>
+                          this.setMasterAZMode(MASTER_AZ_MODE_AUTO)
+                        }
                       />
                     </InputGroup>
                   </RUMActionTarget>
+                  {masterAZMode === MASTER_AZ_MODE_AUTO && (
+                    <p>
+                      An Availabilty Zone will be automatically chosen from the
+                      existing ones.
+                    </p>
+                  )}
                   <RUMActionTarget
                     name={RUMActions.SelectMasterAZSelectionManual}
                   >
                     <InputGroup>
                       <RadioInput
                         id='manual'
-                        checked={hasAZLabels}
+                        checked={masterAZMode === MASTER_AZ_MODE_MANUAL}
                         label='Manual'
-                        onChange={() => this.toggleMasterAZSelector(true)}
+                        onChange={() =>
+                          this.setMasterAZMode(MASTER_AZ_MODE_MANUAL)
+                        }
+                      />
+                    </InputGroup>
+                  </RUMActionTarget>
+                  {masterAZMode === MASTER_AZ_MODE_MANUAL && (
+                    <AZWrapperDiv>
+                      <AvailabilityZonesParser
+                        min={minAZ}
+                        max={maxAZ}
+                        defaultValue={defaultAZ}
+                        zones={this.props.availabilityZones}
+                        updateAZValuesInParent={this.updateAZ}
+                        isLabels={true}
+                        isRadioButtons
+                      />
+                      {zonesArray.length < 1 && (
+                        <p className='danger'>
+                          Please select one availability zone.
+                        </p>
+                      )}
+                      {zonesArray.length > maxAZ && (
+                        <p className='danger'>
+                          {maxAZ} is the maximum you can have. Please uncheck{' '}
+                          {zonesArray.length - maxAZ} of them.
+                        </p>
+                      )}
+                    </AZWrapperDiv>
+                  )}
+                  <RUMActionTarget
```

Comment:
I guess the new option only makes sense on Azure, right?

Can we only render it when the provider is `azure`, similar to what we in the node pool creation form?
https://github.com/giantswarm/happa/blob/c1db97f7b6da7d017da430e69c6d5e829fedcbc9/src/components/Cluster/ClusterDetail/AddNodePool/AddNodePoolAZSelection.tsx#L203

Reply:
done